encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py

```diff
@@ -1,140 +1,144 @@
 import os
-import numpy as np
-import pytest
+from itertools import product
 import random
 import string
-from .. import TransformerTorchEncoder
+
+import numpy as np
+import pytest
 from jina.executors import BaseExecutor
 from jina.executors.metas import get_default_metas
 
-
-@pytest.fixture(scope='function')
-def random_workspace_name():
-    """Generate a random workspace name with digits and letters."""
-    rand = ''.join(random.choices(string.ascii_uppercase + string.digits, k=6))
-    return f'JINA_TEST_WORKSPACE_{rand}'
+from .. import TransformerTorchEncoder
 
 
-@pytest.fixture(scope='function')
-def test_metas(tmpdir, random_workspace_name):
-    os.environ[random_workspace_name] = str(tmpdir)
+@pytest.fixture(scope="function")
+def test_metas(tmp_path):
     metas = get_default_metas()
-    metas['workspace'] = os.environ[random_workspace_name]
-    if 'JINA_TEST_GPU' in os.environ:
-        metas['on_gpu'] = True
+    metas["workspace"] = str(tmp_path)
```

Comment:
same here, keep single quote

Reply:
Will do. I was actually trying to find any reference about the code style, but didn't find anything. Is it available somewhere?